plugins/colorbutton/plugin.js

```diff
@@ -233,7 +233,7 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					};
 
 					editor.focus();
-					editor.applyStyle( new CKEDITOR.style( colorStyle, { color: color } ) );
+					editor[ ( color ? 'apply' : 'remove' ) + 'Style' ]( new CKEDITOR.style( colorStyle, { color: color } ) );
```

Comment:
There's already a `removeStyle` call few lines above, there's no reason to do that twice.